src/app/view/components/listing-data-table/listing-data-table.component.ts

```diff
@@ -17,27 +18,38 @@ export class ListingDataTableComponent extends BaseComponent {
     @Input() columns: DataTableColumn[];
     @Input() url: string;
     @Input() actionHeaderOptions: ActionHeaderOptions;
+    @Input() buttonActions: ButtonAction[] = [];
+    @Input() defaultSortOrder: string;
```

Comment:
I think that this property is to set the default order direction (ASC or DESC), so it should be a 'OrderDirection' not a string